src/plugins/reporter-bugzilla.c

```diff
@@ -312,21 +312,21 @@ int main(int argc, char **argv)
 
     export_abrt_envvars(0);
 
-    map_string_t *settings = new_map_string();
-
-    /* load some defaults from /etc/os-release */
-    char *os_release = load_text_file("/etc/os-release",
-                   DD_LOAD_TEXT_RETURN_NULL_ON_FAILURE | DD_OPEN_FOLLOW);
-    if (os_release != NULL)
+    /* pull in some defaults from os-release */
     {
-        map_string_t *os_release_map = new_map_string();
-        parse_osinfo(os_release, os_release_map);
-        log_debug("Loaded '/etc/os-release'");
-        bug_url = xstrdup(get_map_string_item_or_NULL(os_release_map, "BUG_REPORT_URL"));
-        if (bug_url != NULL)
-            rhbz->b_bugzilla_url = bug_url;
+        problem_data_t *problem_data = create_problem_data_for_reporting(dump_dir_name);
```

Comment:
This approach make sense to me. I have just two comments:
1. dump_dir_name might be NULL here
2. there is no need to call `create_problem_data_for_reporting` twice time